### DISCUSSION TITLE
#1 📑 Weekly Report 1

### DISCUSSION THREAD

**Discussion by ghost** [SE-TINF22B6/Statify#20] [Blog]
**Statify**

ENG:
The Spotify Web API allows developers to access and interact with Spotify's vast music catalog and user-related data. With this API, developers can create applications that can perform tasks like retrieving information about tracks, albums, and artists, managing user playlists, analyzing music content, and generating music recommendations. It enables developers to integrate Spotify's music data and services into their own web and mobile applications, making it easier to build music-related apps and services on top of the Spotify platform.

DEU: 
Dieses Projekt schafft eine innovative Verbindung mit Spotify, einem der weltweit führenden Musik-Streaming-Dienste. In diesem Projekt konzentrieren wir uns auf die Entwicklung einer Frontend-Anwendung, die es ermöglicht, faszinierende und leicht verständliche grafische Statistiken aus den Daten von Spotify anzuzeigen. Unser Ziel ist es, eine intuitive Benutzeroberfläche zu schaffen, die Musikliebhabern und Analysten gleichermaßen die Möglichkeit gibt, tiefere Einblicke in ihre Hörgewohnheiten und die Musikwelt im Allgemeinen zu gewinnen. 

Name       | Rolle
-----------| ------
Arthur      |  Scrum Master/   Entwickler
Cornelius |   Entwickler
Arhant     |   Product   Owner/ Entwickler
Ines          |   Entwickler
Viktoriia   |   Entwickler

**Vision**
The web application allows the connection to a user's Spotify account to access their listening history. It is used to provide statistics about their listening habits, such as most-listened-to genres, artists, and time of day, etc..
**Webtechnologien** HTML, CSS und JavaScript sind die Grundlagen des Webdesigns. Diese Technologien werden verwendet, um die Benutzeroberfläche deiner Anwendung zu erstellen und zu gestalten.
**Spotify Web API** Die Spotify Web API ist eine Programmierschnittstelle, die es dir ermöglicht, auf Spotify-Musikdaten zuzugreifen, wie Wiedergabelisten, Künstlerinformationen und Songdetails. Hierzu wird das Spring Boot Tool genutzt.

- Java
- React
- PostgreSQL
- Spotify Web API

Unsere Sprints werden in Wöchentlichen Blöcken aufgeteilt und werden mit wöchentlichen Sprint-plannings getrennt.

**Comment by Stev3eM** [comment]
Good sum-up of your first setup. Looking forward to your project!
